Px.Utils/ModelBuilders/MetadataEntryKeyBuilder.cs

```diff
@@ -0,0 +1,149 @@
+﻿using PxUtils.Models.Metadata;
+using PxUtils.PxFile;
+
+namespace PxUtils.ModelBuilders
+{
+    /// <summary>
+    /// Class for building <see cref="MetadataEntryKey"/> records from strings.
+    /// </summary>
+    public class MetadataEntryKeyBuilder
+    {
+        private readonly char _langParamStart;
+        private readonly char _langParamEnd;
+
+        private readonly char _spesifierParamStart;
+        private readonly char _spesifierParamEnd;
+
+        private readonly char _stringDelimeter;
+        private readonly char _listSeparator;
+
+        private readonly char[] _illlegalKeyTokes;
```

Comment:
Typo with three l's in "_illegalKeyTokes". Also probably supposed to be "tokens"?